indra/llui/llfloater.cpp

```diff
@@ -3623,6 +3623,10 @@ void LLFloater::applyRelativePosition()
 	translate(new_center.mX - cur_center.mX, new_center.mY - cur_center.mY);
 }
 
+bool LLFloater::isDefaultBtnName(const std::string& name)
```

Comment:
You're right. I used it to skip assigning callbacks to default buttons (for example 'minimize'), but then reconsidered this decision as it may be useful to handle such clicks. Removing `isDefaultBtnName`.